### PR TITLE
fix: Add shortcut keys to select next/previous graph

### LINKED ISSUE #37: bug: Add shortcuts to select next/previous graph
Hi,
I now realize that I've miss-understood what Mr. Roussi was saying and what he was asking for.
I heard him say Graph which I interpreted as "The Window" that hold all of the Graphs.
But what he was talking about was the sub-Graphs that are within the main file.
That is, I was thinking that he wants to use Shift+Tab to go from one Graph window to another; like this...
 
Shift+Tab vx

But what he's thinking is the Shift-Tab feature that provided movement from one Plot window to another to another.
So he is raising a valid point in that the current Shift-Tab function is completely incorrect.
As I noted previously, in 2018 these 2 Key sequences were changed...
        -Modified shortcut keys for Shift-Tab and Alt-Tab.
           . Alt-Tab has been removed from usage.
           . Shift-Tab now selects the previous segment.
I don't remember why Alt+Tab was removed from usage.
It currently causes SA to show files on the desktop; which is of no use to us.
The Shift-Tab feature was changed to provide the convention of using tabs to move left and right: Tab and Shift+Tab.
Making this change broke the "Select a new active graph.  Shift+Tab" feature, which never got fixed.
Since Tab and Shift+Tab have been in use for such a long time, I suggest they be left as is.
I suggest leaving the "Select another file (if open).  Ctrl+Tab" feature as is since it will Tab to a new open file with-in SA.
Since there is currently no way to move between multiples Graphs in a single file, as Mr. Roussi is asking for, it seems reasonable to "fix" this issue someday.
Since Alt+Tab is not useful, I suggest connecting it instead to the Active Graph windows code that already exists, and as a result it will become the way to move between multiples Graph. 
Since Shift+Tab is not currently working for his use, his only choice it to use the mouse to select a sub-window.
I have no idea how high a priority this should be, since it may be in little use now-a-days.

Sorry for not figuring all of this out sooner,
Terry

### PR DESCRIPTION
**Pull request by darcywong00**
Fixes #37 

This adds shortcut keys <kbd>Ctrl</kbd>+<kbd>Down</kbd> and <kbd>Ctrl</kbd>+<kbd>Up</kbd> to select the next/previous graph. If the first/last graph is already selected, will wrap around.

These shortcuts are also added to the "Edit" menu list
select graph

TODO on future PR
* Update the SA help (.chm) to include these new shortcuts

### CODE REVIEW

File: Src/SA/SA_View.cpp [hunk 1]
```diff
@@ -363,6 +363,7 @@ BEGIN_MESSAGE_MAP(CSaView, CView)
 	ON_COMMAND(ID_LEGEND_RAWDATA, OnLegendRawdata)
 	ON_COMMAND(ID_MOVE_STOP_CURSOR_HERE, OnMoveStopCursorHere)
 	ON_COMMAND(ID_NEXT_GRAPH, OnNextGraph)
```
Comment (darcywong00) [inline Src/SA/SA_View.cpp:365]: The existing command likely used to happen with a <kbd>Shift</kbd>+<kbd>Tab</kbd> shortcut.

**Comment by megahirt** [comment]
@darcywong00 any ideas how to generate a new chm help file?  Where is the source?

File: Src/SA/SA_View.cpp (hunk 1 above)
Comment (megahirt) [reply Src/SA/SA_View.cpp:365]: nice that there was already a function in existence to show you how to do it :)

**Review by megahirt — APPROVED** [review]
LGTM

**Comment by darcywong00** [comment]
> @darcywong00 any ideas how to generate a new chm help file? Where is the source?

Sorry, I haven't seen the source for the help files

**Pull request merged**